core/editor/src/document/document_message_handler.rs

```diff
@@ -54,6 +54,7 @@ pub enum DocumentMessage {
 	WheelCanvasZoom,
 	SetCanvasRotation(f64),
 	NudgeSelectedLayers(f64, f64),
+	MoveSelectedLayer(i32),
```

Comment:
"Move" sounds like it means *translate* to me. Would you please rename variables/functions/types like "move" or "position" to "rearrange" or "reorder"?

Reply:
Agreed, I think "reorder" is a good term for it.